package.json

```diff
@@ -16,9 +16,29 @@
   "main": "lib/module.js",
   "scripts": {
     "lint": "eslint --ext .vue,.js lib test",
-    "release": "standard-version && git push --follow-tags && npm publish",
+    "release": "release-it",
+    "release-legacy": "standard-version && git push --follow-tags && npm publish",
```

Comment:
Is it required to keep `standard-version` and legacy script? IMO if anything goes wrong we can simply revert back :)

Reply:
I'm fine with removing. I thought I might annoy other maintainers if I force a new way on them.